### ISSUE TITLE
Intel Iris Plus Graphics Crash when Vulkan Enabled

### ISSUE THREAD

**Issue by 47MWCSBN** [nipkownix/re4_tweaks#495]
### Validation

- [X] Game has been updated to the latest RE4HD release (v1.1: https://www.re4hd.com/?p=9552)
- [X] Data has been [validated with SFV](https://www.re4hd.com/?p=9454) (a guide to using QuickSFV is available at that link, use that with this 1.1 SFV: [BIO4-HDProject1.1-SFV.zip](https://github.com/emoose/re4-research/files/9207370/BIO4-HDProject1.1-SFV.zip))

### Describe your issue here (drag+drop ZIP to attach it)

dont know if this related to Intel UHD issue, but my IGP cannot run Vulkan. when i try disabled it the game running fine :(

**Comment by nipkownix** [comment]
Yeah, I think DXVK has some issues with Intel IGPs, but I'm not sure. I've recently updated DXVK, could you see if this build improves anything?

https://github.com/nipkownix/re4_tweaks/suites/11649125646/artifacts/604985079

**Comment by 47MWCSBN** [comment]
@nipkownix Nope, still crash :(

**Comment by 47MWCSBN** [comment]
@nipkownix btw can you fix or change Vsync in game somehow?

**Comment by nipkownix** [comment]
Hm? Vsync works here. It doesn't for you?

**Comment by 47MWCSBN** [comment]
Vsync make slow motion movement. try disabled it, but it soo jitter make me sick

**Comment by nipkownix** [comment]
What refresh rate is your monitor? Please enabled "VerboseLog" in the .ini file, run the game again, and then upload the .log here.

**Comment by 47MWCSBN** [comment]
My Refresh rate is 60 Hz. the slowdown occur during fog or particle, the only fix for me is just disabled vsync
